### ISSUE TITLE
FeaturesAnalysis promos should only output as a list if more than one promo

### ISSUE THREAD

**Issue by mykeels** [bbc/simorgh#6045]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behaviour:

1. Create a story where only one item is passed to the FeaturesAnalysis container component
2. Note that the produced DOM structure contains a `<ul>` tag. 

**Expected behaviour**
If the component only contains one item, the StoryPromo component should be rendered directly without a wrapping `<ul>` element.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
